resources/js/index.js

```diff
@@ -47,3 +46,29 @@ function toggle_complete(element) {
   });
   sessionStorage.setItem("list", list_task);
 }
+
+function todo_list() {
+  let element = save_task();
+  let string_to_date = moment(element.due_date).format();
+  const formatted_due_date = moment(string_to_date).format("L");
+  const formatted_creation_date = moment(element.creation_date).format("L");
+  const listContainer = document.getElementById("list1");
+
+  listContainer.insertAdjacentHTML(
```

Comment:
First time I see an adjancentHTML 😮

Reply:
That method works for add new content to element in DOM, I remark "ADD"